it/pages/LanguageSpec.scala

```diff
@@ -69,7 +66,7 @@ class LanguageSpec extends IntegrationBaseSpec {
 
     "language preference is not set in the cookie" should {
```

Comment:
This test is now incorrect as the language is being set to english by default in the cookie. May be best remedied by having lang as optional string and a small bit of logic in the cookie baker?